.github/workflows/avalon.yml

```diff
@@ -16,14 +16,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm install hathora -g
-      - run: hathora cloud deploy --appName hathora-avalon --token ${{ secrets.HATHORA_TOKEN }}
+      - run: npm ci
+      - run: npm install ts-node -g
+      - run: ts-node src/cli.ts cloud deploy --appName hathora-avalon --token ${{ secrets.HATHORA_TOKEN }}
   frontend:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm install hathora surge -g
-      - run: COORDINATOR_HOST=coordinator.hathora.com APP_SECRET=${{ secrets.AVALON_APP_SECRET }} hathora build --only client
+      - run: npm install ts-node surge -g
```

Comment:
I think we should be using the released versions of Hathora in the example games